packages/app/components/Queue/TA/TAQueueList.tsx

```diff
@@ -259,16 +262,46 @@ const NoQuestionsText = styled.div`
   color: #212934;
 `;
 
+function NotifReminderButton() {
+  const [isNotifOpen, setIsNotifOpen] = useState(false);
+  const NotifRemindText = styled(Button)`
+    margin-top: 16px;
+    border-radius: 6px;
+    border: 0;
+    background: #fff;
+  `;
+
+  return (
+    <>
+      <NotifRemindText onClick={(e) => setIsNotifOpen(true)}>
```

Comment:
I would rename this styled component to `NotifRemindButton` since it's a styled Button